### PR TITLE
Fix/zy/old test center reapply error

### PR DESCRIPTION
**Pull request by ssignik**
1.old test center reapply error fix with user insensible

### CODE REVIEW

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse" border=1>
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>Check代码检查</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/certification-server/3/3_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>DT覆盖率</td>
    <td>7.3953%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/certification-server/3/3_测试覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/certification-server/3/3_敏感信息扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>漏洞扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/certification-server/3/3_漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>安全编码扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/certification-server/3/3_安全编码扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>TestPlan接口测试</td>
    <td>&#9989;</td>
    <td></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/128c1a12cee14cd3ae4059676f9c3c61/b15d460aac00471db8176dc385c26a7a">点击跳转</a></td>
</tr>
</table>

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Review by shishupei — APPROVED** [review]
注意编码规范

File: openeulerSever/src/test/java/com/huawei/it/euler/model/enumeration/CenterEnumTest.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        allInnovationCenter.add(CenterEnum.OSV_USER);
+
+        List<CenterEnum> allCenters = CenterEnum.getAllCenters();
+
+        Assertions.assertEquals(allInnovationCenter, allCenters);
+    }
+
+    @Test
+    @DisplayName("根据新name查找")
+    public void testGetAll2(){
```
Comment (ssignik) [inline openeulerSever/src/test/java/com/huawei/it/euler/model/enumeration/CenterEnumTest.java:76]: 该方法命名要清晰

File: openeulerSever/src/main/java/com/huawei/it/euler/model/enumeration/CenterEnum.java [hunk 2]
```diff
@@ -53,7 +65,10 @@ public enum CenterEnum {
 
     static {
         for (CenterEnum status : values()) {
-            CENTER_ENUM_MAP.put(status.getId(), status);
+            if (status != OLD_USER && status != OLD_CHINA_REGION && status != OLD_SIG_GROUP
+                    && status != OLD_EULER_IC && status != OLD_FLAG_STORE && status != OLD_ADMIN) {
```
Comment (ssignik) [inline openeulerSever/src/main/java/com/huawei/it/euler/model/enumeration/CenterEnum.java:69]: 明确筛选哪些字段而非不筛选哪些，意图会更加清晰

File: openeulerSever/src/test/java/com/huawei/it/euler/model/enumeration/CenterEnumTest.java (hunk 1 above)
Comment (ssignik) [reply openeulerSever/src/test/java/com/huawei/it/euler/model/enumeration/CenterEnumTest.java:76]: 清楚

File: openeulerSever/src/main/java/com/huawei/it/euler/model/enumeration/CenterEnum.java (hunk 2 above)
Comment (ssignik) [reply openeulerSever/src/main/java/com/huawei/it/euler/model/enumeration/CenterEnum.java:69]: 明白

**Pull request merged**